src/components/page/Footer/footer.css

```diff
@@ -0,0 +1,15 @@
+.reveal-floating-logo {
+  position: fixed;
+  bottom: 0;
+  left: 10px;
+}
+
+.reveal-floating-logo img {
+  width: 100px !important;
+  height: auto;
+}
+
+.floating-logo-text {
+  vertical-align: bottom;
+  line-height: 45px;
+}
```

Comment:
Was there a requirement to make this thing float around?  I don't like that it does, and it just makes the page more busy without adding value other than "hey see our nice shiny logo".